sql-aggregates/starring-in.sql

```diff
@@ -0,0 +1,8 @@
+select "genres"."name" as "genre",
+       count("filmGenre".*) as "appearedCount"
+  from "genres"
+  join "filmGenre" using ("genreId")
+  join "castMembers" using ("filmId")
+  join "actors" using ("actorId")
+  where "actors"."firstName" = 'Lisa'
+group by "genres"."name";
```

Comment:
Regardless of the current contents of the database, your query should also restrict actors to those with the last name of `"Munroe"`:

---

![image](https://user-images.githubusercontent.com/16143771/180911340-7e984c53-2d72-4d4d-9952-d579887735fb.png)

Reply:
![image](https://user-images.githubusercontent.com/90783131/181141604-10c0361e-391c-4c28-96d7-782bc3b40b4e.png)